q2_feature_table/plugin_setup.py

```diff
@@ -335,17 +337,20 @@
                  'that must be met to be included in the filtered feature '
                  'table. If not provided, all features in `metadata` that are '
                  'also in the sequences will be retained.',
-        'exclude_ids': 'If true, the features selected by `metadata` or '
-                       '`where` parameters will be excluded from the filtered '
-                       'sequences instead of being retained.'
+        'exclude_ids': 'If true, the features selected by `metadata`, `table`,'
```

Comment:
How about:

```
If true, the features selected by the `metadata` (with or without the `where` parameter)
or `table` parameters will be excluded from the filtered sequences instead of being retained.
```